src/bundles/ApplicationsAdmin/components/EvidenceAssessmentList/EvidenceAssessmentList.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import format from 'date-fns/format'
+import './EvidenceAssessmentList.css'
+
+class EvidenceAssessmentList extends React.Component {
+  render() {
+    const { evidence } = this.props
+
+    if (evidence && evidence.length > 0) {
+      return (
+        <div styleName="evidenceList">
+          <h1 className="au-display-xl">Submitted evidence assessments</h1>
+          <table className="content-table">
+            <thead>
+              <tr>
+                <th>submitted_at</th>
+                <th>supplier</th>
+                <th>brief</th>
+                <th>domain</th>
+                <th>actions</th>
+              </tr>
+            </thead>
+            <tbody>
+              {evidence.map(e => (
+                <tr key={e.id}>
+                  <td>{format(e.submitted_at, 'DD-MM-YYYY')}</td>
+                  <td>{e.supplier_name}</td>
+                  <td>
+                    {e.brief_id && (
+                      <a href={`https://marketplace.service.gov.au/2/digital-marketplace/opportunities/${e.brief_id}`}>
```

Comment:
hard coded url?

Reply:
There is no straight forward way from the admin frontend to detect which environment it is running in from what I could tell (e.g. even if we can detect it isn't prod, how do we know which of our rc/dev/alpha etc environments is it?)

Reply:
Updated to use the env var value.